frontend/src/components/VideoCall/VideoFrontend/components/PollsWindow/PollCard/PollCard.tsx

```diff
@@ -58,15 +68,26 @@ const useStyles = makeStyles({
     gridRow: '2 / span 1',
     gridColumn: '2 / span 1',
   },
+  deleteButton: {
+    margin: '0.5rem',
+    justifySelf: 'end',
+    flexDirection: 'column',
+    float: 'right',
+    gridRow: '1 / span 1',
+    gridColumn: '2 / span 1',
+  },
```

Comment:
could you change line 56 from `padding: '1em 1em 0.1em 0.8em',` to `padding: '1em 4em 0.1em 0.8em',`? just to prevent overlap between the delete button and the question title